README.adoc

```diff
@@ -0,0 +1,203 @@
+= Secure Xamarin Template
+
+image:https://img.shields.io/badge/Android-Compatible-green.svg[]
+image:https://img.shields.io/badge/iOS-Compatible-green.svg[]
+
+[width="85%"]
+|===========================================================================================================
+|                 | *Project Info*
+| License:        | Apache License, Version 2.0
+| Build:          | msbuild
+| End User Documentation:  | link:https://docs.aerogear.org[]
+| Community Documentation: | link:https://aerogear.org[]
+| Issue tracker:  | link:https://issues.jboss.org/browse/AEROGEAR[]
+| Mailing lists:  | link:https://groups.google.com/forum/#!forum/aerogear[aerogear-dev]
+|===========================================================================================================
+
+A Xamarin Application that demonstrates features of the Aerogear Xamarin SDK on iOS and Android.
+
+For more information, please check the https://github.com/feedhenry/mobile-security[Mobile Security Project].
+
+== Features
+- OpenID Connect Authentication
+- Single Sign-on
+- Device Trust Checks (Jailbreak/ROOT, etc)
+- 2FA with OTP (via Keycloak)
+- Authentication Brute Force Detection (via Keycloak)
+- Account Lockout Policies (via Keycloak)
+- Authentication/Access Control Auditing & Logging (via Keycloak)
+- Authenticated Calls to Protected Endpoints (via Keycloak)
+
+== Get the sources
+The `master` branch will always track to the latest release of the SDK.
+
+[source, bash]
+----
+$ git clone git@github.com:aerogear/xamarin-showcase-template.git
+$ cd xamarin-showcase-template
+----
+
+== Run the application
+
+A Xamarin application can run on both iOS and Android.
+
+=== Run on iOS Simulator with Visual Studio
+
+[source, bash]
+----
+$ open xamarin-showcase-template.sln
+----
+
+- A Visual Studio window will open. In the top bar, choose Example.iOS->Debug->iPhone 8 iOS 11.4.
```

Comment:
I think there should be a pre-req section and list all the SDKs/IDEs the user should install?

Reply:
👍